src/pages/Main/FilterButton/FilterModal/FilterModal.js

```diff
@@ -0,0 +1,466 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+function Modal({ title, handleModal, tagFetchHandler }) {
+  const [filterCategories, setFilterCategories] = useState([]);
+  const [selectCategory, setselectCategory] = useState(1);
+  const [multipleAlert, setMultipleAlert] = useState(false);
+
+  useEffect(
+    () =>
+      setFilterCategories(
+        fetch('http://54.180.99.36:8000/jobpostings/tags')
+          .then(res => res.json())
+          .then(data => {
+            setFilterCategories(
+              data.result.map(({ id, name, is_multiple_choice, tags }) => ({
+                id,
+                name,
+                is_multiple_choice,
+                tags,
+                selected: [],
+              }))
+            );
+            console.log(`data`, data);
+          })
+      ),
+    []
+  );
+
+  useEffect(() => setMultipleAlert(false), [selectCategory]);
+
+  const tagListHandler = id => setselectCategory(id);
+
+  const prevSelected = id => {
+    const prevSelected = filterCategories.filter(
+      filterList => filterList.id === id
+    )[0].selected;
+    return prevSelected;
```

Comment:
굳이 따로 변수 선언해서 리턴할 필요 없어보입니다.
```suggestion
     return filterCategories.filter(
      filterList => filterList.id === id
    )[0].selected;
```

Reply:
뭔가 고정관념이 있었던거 같내요!